packages/spruce-skill-server/tests/GQLMethodTests.js

```diff
@@ -0,0 +1,76 @@
+const assert = require('chai').assert
+const SpruceTest = require('./SpruceTest')
+const config = require('config')
+const faker = require('faker')
+
+class GQLMethodTests extends SpruceTest(`${__dirname}/../../spruce-skill/`) {
+	setup() {
+		it('Can call query', () => this.doQuery())
+		it('Can not call query as mutation', () => this.doQueryAsMutate())
+		it('Can call mutation', () => this.doMutate())
+		it('Can not call mutation as query', () => this.doMutateAsQuery())
+	}
+
+	async doQuery() {
+		const result = await this.ctx.sb.query(`
```

Comment:
Use `graphql-tag` to tag these template literals and get consistent formatting. (Not sure if it's included by default, been a few weeks. If not don't worry about it)

Reply:
@gcpantazis This method actually doesn't support using `graphql-tag` since it's making the api request directly (not using apollo client).

A future enhancement could be that we support both ways?  And/or we move to using the apollo client for backend gql api requests